Dockerfile

```diff
@@ -132,7 +187,7 @@ RUN set -x \
     && apt-get update \
     && apt-get install --no-install-recommends -y \
         libpq5 libxml2 libxslt1.1 libcurl4  \
-        $(if [ "$DEVEL" = "yes" ]; then echo 'bash libjpeg62 postgresql-client build-essential libffi-dev libxml2-dev libxslt-dev libpq-dev libcurl4-openssl-dev libssl-dev git libcairo2-dev libfreetype6-dev libjpeg-dev libpng-dev libz-dev'; fi) \
```

Comment:
When the blog was added, this is the list of deps added for it to work:
git libcairo2-dev libfreetype6-dev libjpeg-dev libpng-dev libz-dev

I think all of them should migrate to the new docs target instead, right?

Reply:
ty.